src/features/profile/Profile.tsx

```diff
@@ -0,0 +1,186 @@
+import { FC } from 'react';
+import { Link, useLocation, useParams } from 'react-router-dom';
+import { Button } from '../../common/components';
+import NarrativeList from '../../common/components/NarrativeList/NarrativeList';
+import {
+  useAppDispatch,
+  useAppSelector,
+  usePageTitle,
+} from '../../common/hooks';
+import { authUsername } from '../auth/authSlice';
+import PageNotFound from '../layout/PageNotFound';
+import {
+  cache,
+  clearError,
+  error,
+  loadProfile,
+  selectUsername,
+  usernameSelected,
+} from './profileSlice';
+import classes from './Profile.module.scss';
+
+export const ProfileNarrativesMessage: FC<{
+  realname: string;
+  yours: boolean;
+}> = ({ realname, yours }) => {
+  if (yours) {
+    return <span>This table shows all of your narratives.</span>;
+  }
+  return (
```

Comment:
I think that all of these small components (`ProfileNarrativeMessage`, `ProfileResume`, `ProfileInfobox`, etc.) should be folded into the `Profile` component. when you just have nested components inside of components that effectively render only DOM elements, you lose any of the advantages of breaking things down into subcomponents and start obfuscating what the actual structure of the html itself is. I just think its easier for developers down the road to follow the HTML

Reply:
Added an explanatory comment in the latest commit.